tests/acceptance/WPFirstCest.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+
+class WPFirstCest
```

Comment:
That name is just a custom one when I did the installation suite. It's just a custom test to see if it works. The name is based off how I generated the file. You can generate tests on the command line and give them custom values.